src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,10 +6,12 @@
 public class CliSyntax {
 
     /* Prefix definitions */
+    public static final Prefix PREFIX_PID = new Prefix("id/");
     public static final Prefix PREFIX_NAME = new Prefix("n/");
-    public static final Prefix PREFIX_PHONE = new Prefix("p/");
-    public static final Prefix PREFIX_EMAIL = new Prefix("e/");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
+    public static final Prefix PREFIX_PRENAME = new Prefix("p/");
```

Comment:
```suggestion
    public static final Prefix PREFIX_PREFERRED_NAME = new Prefix("p/");
```

Maybe spell it out fully so that its easier to know and remember what this Prefix is for

Reply:
Have been updated to `PREFIX_PREFERRED_NAME`